treeherder/webapp/api/seta.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework import (serializers,
+                            status,
+                            viewsets)
+from rest_framework.response import Response
+
+from treeherder.seta.analyze_failures import get_failures_fixed_by_commit
+from treeherder.seta.job_priorities import (SetaError,
+                                            seta_job_scheduling)
+from treeherder.seta.models import JobPriority
+from treeherder.seta.seta import get_distinct_tuples
+
+
+class SetaJobPrioritySerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = JobPriority
+
+
+class SetaJobPriorityViewSet(viewsets.ViewSet):
+    def list(self, request, project):
+        ''' Routing to /api/project/{project}/seta/v1/job-priorities/
+
+        This API can potentially have these consumers:
+            * Buildbot
+              * build_system_type=buildbot
+              * priority=5
+              * format=json
+            * TaskCluster (Gecko decision task)
+              * build_system_type=taskcluster
+              * format=json
+              * user agent == 'TaskCluster'
+        '''
+        # XXX: At the moment we're not taking advantage of "per-project" approach since
```

Comment:
I don't think we want SETA to be per project- the idea is that in a given time window (90 days), we have changes affecting our branches and specific tests which provide a lot of value- there might be some arguments for a project branch to have different SETA data, but I don't think overall we would want per branch data sets.